dllmain.cpp

```diff
@@ -472,6 +472,14 @@ void DelayedStart()
 		SetSingleCoreAffinity();
 	}
 
+	FindGetModelID();
```

Comment:
Shouldn't this be inside the `if (SpecularFix)` statement?

Reply:
Better yet, what if you added this line to the `PatchSpecular()` function and then you can remove the line for this out of the `patches.h` file.

Reply:
@PolyMega [explains it here](https://github.com/elishacloud/Silent-Hill-2-Enhancements/issues/155#issuecomment-738350704), the [old method](https://github.com/elishacloud/Silent-Hill-2-Enhancements/blob/90c8cf479bdcec768126f0f50cb221e90f7c6603/Wrappers/d3d8/IDirect3DDevice8.cpp#L1054) of detecting the Woodside Room 208 TV static (based on checking texture dimensions) stops working when `IncreaseNoiseEffectRes` is active.

Using `GetModelID()` makes this really easy to solve, but it means we need to call `FindGetModelID()` even when `SpecularFix` is disabled. I could make it so it's only called by dependent fixes (Just `SpecularFix` and `EnableSoftShadows` at the moment) if you'd prefer that?

Reply:
> I could make it so it's only called by dependent fixes (Just `SpecularFix` and `EnableSoftShadows` at the moment) if you'd prefer that?

Yes, that sounds good.  I try to make things be able to be disabled if possible.

Reply:
How's that? I just remembered `EnableSoftShadows` has no "setup" function to put it in because it's not a patch.

Reply:
Looks good.